### DISCUSSION TITLE
easy to build single executable like what nicegui does?

### DISCUSSION THREAD

**Discussion by laoshaw** [reflex-dev/reflex#1294] [General]
NiceGUI has one flag ( ui.run(native=True) )to turn itself into a single exe, can pynecone do it now? it's very useful sometimes to ship the GUI to others who does not have python installed.

**Comment by masenf** [comment]
Currently reflex (formerly pynecone) is focused on shipping web apps. Due to the dependencies (unzip and nodejs), there's not currently a good path to making a self contained executable. We are discussing internal the prospect of automatically installing a working copy of nodejs if one is not present, which might allow a workflow such as this.

**Comment by qdaicho** [comment]
Could you somehow have it run on tauri? Tauri is the next electron from what I hear.

**Comment by sandyowlet** [comment]
Hi, I just created a repository that demonstrates how to use pyinstaller and pywebview to package Reflex apps as executables:
https://github.com/sandyowlet/reflex-desktop-app
